datasets/simple_questions_v2/simple_questions_v2.py

```diff
@@ -0,0 +1,168 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+SimpleQuestions is a dataset for simple QA, which consists
+of a total of 108,442 questions written in natural language by human
+English-speaking annotators each paired with a corresponding fact,
+formatted as (subject, relationship, object), that provides the answer
+but also a complete explanation.  Fast have been extracted from the
+Knowledge Base Freebase (freebase.com).  We randomly shuffle these
+questions and use 70% of them (75910) as training set, 10% as
+validation set (10845), and the remaining 20% as test set.
+"""
+_HOMEPAGE_URL = "https://research.fb.com/downloads/babi/"
+_CITATION = """\
+@misc{bordes2015largescale,
+      title={Large-scale Simple Question Answering with Memory Networks},
+      author={Antoine Bordes and Nicolas Usunier and Sumit Chopra and Jason Weston},
+      year={2015},
+      eprint={1506.02075},
+      archivePrefix={arXiv},
+      primaryClass={cs.LG}
+}
+"""
+
+_URL = "https://uc7042fc5e1d2d9c831a318b49e7.dl.dropboxusercontent.com/cd/0/get/BEVJ4i_TOAt1pM8VdyJa0ctwmjEerlbDzNw2u8wXoMjY4s6WbgiKd3CA3gJ1deDcCR-5yyUKdpL0zS76uTuGcjX8To48_vwLjyeJwej8sbVBlg/file?_download_id=6436861371570024931106653846455288758893761540515299505641721092&_notify_domain=www.dropbox.com&dl=1"
```

Comment:
Could you use this url instead ?
```
https://www.dropbox.com/s/tohrsllcfy7rch4/SimpleQuestions_v2.tgz?dl=1
```
I feel like the current one might have a limited lifetime

After updating the url you'll have to regenerate the dataset_infos.json